lib/ts/unit-tests/test_layout.cpp

```diff
@@ -0,0 +1,80 @@
+/** @file
+  Test file for layout structure
+  @section license License
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include "catch.hpp"
+
+#include "string_view.h"
+#include "I_Layout.h"
+#include "ink_platform.h"
+#include <string>
+
+using namespace std;
+
```

Comment:
you might want to have a test the show the value given TS_ROOT, TS_RUNROOT, TS_ROOT and TS_RUNROOT are set to a value.